tests/functional/UseCase/Table/ExportTableToFileTest.php

```diff
@@ -287,6 +289,128 @@ public function testExportTableToSlicedFile(bool $isCompressed, array $expectedF
         $db->close();
     }
 
+    public function testExportTableToFileLimitColumns(): void
+    {
+        $bucketDatabaseName = $this->bucketResponse->getCreateBucketObjectName();
+        $sourceTableName = md5($this->getName()) . '_Test_table_export';
+        $exportDir = sprintf(
+            'export/%s/',
+            str_replace([' ', '"', '\''], ['-', '_', '_'], $this->getName())
+        );
+
+        // create table
+        $db = $this->getConnection($this->projectCredentials);
+        $sourceTableDef = $this->createSourceTable($bucketDatabaseName, $sourceTableName, $db);
+
+        // clear files
+        $s3Client = $this->getS3Client(
+            (string) getenv('AWS_ACCESS_KEY_ID'),
+            (string) getenv('AWS_SECRET_ACCESS_KEY'),
+            (string) getenv('AWS_REGION')
+        );
+        $this->clearS3BucketDir(
+            $s3Client,
+            (string) getenv('AWS_S3_BUCKET'),
+            $exportDir
+        );
+
+        // export command
+        $cmd = new TableExportToFileCommand();
+
+        $path = new RepeatedField(GPBType::STRING);
+        $path[] = $bucketDatabaseName;
+        $cmd->setSource(
+            (new ImportExportShared\Table())
+                ->setPath($path)
+                ->setTableName($sourceTableName)
+        );
+        $cmd->setFileProvider(FileProvider::S3);
+        $cmd->setFileFormat(FileFormat::CSV);
+
+        $columnsToExport = new RepeatedField(GPBType::STRING);
+        $columnsToExport[] = 'col1';
+        $columnsToExport[] = 'col2';
```

Comment:
nechceme to jeste otestovat pro UC bez nastavenych columns?

Reply:
no šak ten test tam už byl předtím.

Reply:
`testExportTableToFile`

Reply:
jo, jasne, implicitne to uz mame pokryte , ok